libs/wire-api/src/Wire/API/Routes/MultiVerb.hs

```diff
@@ -244,11 +244,21 @@ instance KnownStatus s => IsResponse cs (RespondAs '() s desc ()) where
     guard (responseStatusCode output == statusVal (Proxy @s))
 
 instance
-  (KnownStatus s, KnownSymbol desc, S.ToSchema a) =>
+  (KnownSymbol desc, S.ToSchema a) =>
   IsSwaggerResponse (RespondAs ct s desc a)
   where
   responseSwagger = simpleResponseSwagger @a @desc
 
+instance
+  {-# OVERLAPPING #-}
```

Comment:
If we specify the kind of `ct` in  the other instance (with `ct :: *`), then the overlapping pragma should is not necessary.